### PR TITLE
Bagatur/docs last updated

### PR DESCRIPTION
**Pull request by baskaryan**

### CODE REVIEW

**Comment by nfcampos** [comment]
Some issues
- missing here https://langchain-git-bagatur-docslastupdated-langchain.vercel.app/docs/expression_language/interface
- wrong formatting here https://langchain-git-bagatur-docslastupdated-langchain.vercel.app/docs/expression_language/why
- running into next line https://langchain-git-bagatur-docslastupdated-langchain.vercel.app/docs/expression_language/get_started
Also they all say yesterday not sure if that's expected?

**Comment by baskaryan** [comment]
> Some issues
> 
> * missing here https://langchain-git-bagatur-docslastupdated-langchain.vercel.app/docs/expression_language/interface
> * wrong formatting here https://langchain-git-bagatur-docslastupdated-langchain.vercel.app/docs/expression_language/why
> * running into next line https://langchain-git-bagatur-docslastupdated-langchain.vercel.app/docs/expression_language/get_started
> 
> Also they all say yesterday not sure if that's expected?

not sure what's going on with the dates, works locally so must be something with how vercel checks out repo

**Comment by efriis** [comment]
Could we do this in the frontmatter and display with `showLastUpdatedTime` instead?

https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#last_update

https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#showLastUpdateTime

File: docs/vercel_requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@
 -e ../libs/community
 -e ../libs/core
 urllib3==1.26.18
+nbformat
```
Comment (efriis) [inline docs/vercel_requirements.txt:5]: I don't think this is needed at build time, so doesn't need to be in vercel_requirements right?

**Comment by baskaryan** [comment]
> Could we do this in the frontmatter and display with `showLastUpdatedTime` instead?
> 
> https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#last_update
> 
> https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#showLastUpdateTime

doesn't seem to work, maybe because of how we're converting notebooks https://github.com/langchain-ai/langchain/pull/15402

**Pull request closed without merging**